packages/box/lib/utils/unbox.ts

```diff
@@ -37,12 +37,15 @@ async function verifyURL(url: string) {
 }
 
 function fetchRepository(url: string, dir: string) {
-  return new Promise((accept, reject) =>
-    // Download the package from github.
-    ghdownload(url, dir)
-      .on("err", reject)
-      .on("end", accept)
-  );
+  return new Promise((accept, reject) => {
+    // Download the package from github - download(url, dir, options, cb)
+    download(url, dir, error => {
```

Comment:
want to just `promisify` this, instead of creating a `new Promise()` manually?

Reply:
Yeah sure, let's do it!